src/Game/zMusic.h

```diff
@@ -3,30 +3,27 @@
 
 #include <types.h>
 
-struct zVolumeInfo {
-	float32 cur;
-	float32 end;
-	float32 inc;
-	uint32 adjusted[2];
+struct zVolumeInfo
+{
+    float32 cur;
+    float32 end;
+    float32 inc;
+    uint32 adjusted[2];
 };
 
-struct zMusicTrackInfo {
-	uint32 snd_id;
-	uint32 loop;
-	int32 unk;
-	uint32 assetID;
-	float32 lastVol;
-};
-
-enum LevelMusic {
-	//TODO: POPULATE MUSIC LISTING!
-	Null
+struct zMusicTrackInfo
+{
+    uint32 snd_id;
+    uint32 loop;
+    int32 situation; // Is actually a zMusicSituation*.
```

Comment:
I know it's a pointer and 4 bytes anyway, but if copying dwarf struct definitions into the source code, please copy them as they are without changing any types. Sometimes this is annoying because you have to go down a rabbit hole of defining structs, but it will be helpful to others down the road if we import everything as we come across it.

https://github.com/pslehisl/bfbbdecomp/blob/7437353b1ae66044e80bed5c02704a234ca25609/dwarf/SB/Game/zMusic.cpp#L1949-L1962

This struct is easy to import.